docs/source/models/gbd2017/causes/ischemic_stroke/ischemic_stroke.rst

```diff
@@ -0,0 +1,68 @@
+.. _2017_cause_ischemic_stroke:
+
+===================
+Ischemic Stroke
+===================
+
+Disease Description
+-------------------
+
+.. todo::
+
+   Add more information and references. In particular, find data about global prevalence and relation to disease fatal and non-fatal description.
+
+
+
+
+
+Modeling Ischemic Stroke in GBD 2017
+-------------------------------------------
+
+Strokes in GBD 2017
+++++++++++++++++++++++++++++++++++++++
+
+
+Modeling Strategy for Ischemic Stroke
+++++++++++++++++++++++++++++++++++++++++++++
+
+Cause Model Diagram
+--------------------------
+.. image:: ischemic_stroke_states.png
```

Comment:
Consider standardizing this to look more like some of the others--circles instead of rectangles, and labelled with the shorthand state name, to be referenced in the data input tables.

Reply:
Oops that's not actually accurate---it looks like Nathaniel's neural tube defects diagram is between the format you have and the one I described. Maybe we can confirm during coworking that we don't need to standardize!

Reply:
My old neural tube defects diagram had lots of words in the state bubbles, but in the new one, each state is labeled with a single letter (new diagram was merged into master on Friday).